src/main/resources/swagger/api-docs

```diff
@@ -0,0 +1,28 @@
+{
+  "apiVersion": "1.0.0",
+  "swaggerVersion": "1.2",
+  "apis": [{
+    "path": "https://rawls-dev.broadinstitute.org/listings/authentication",
+    "description": "Google authentication API"
+  }, {
+    "path": "https://rawls-dev.broadinstitute.org/listings/submissions",
+    "description": "Submissions API"
+  }, {
+    "path": "https://rawls-dev.broadinstitute.org/listings/entities",
+    "description": "Entity manipulation API"
+  }, {
+    "path": "https://rawls-dev.broadinstitute.org/listings/methodconfigs",
+    "description": "Method Configuration manipulation API"
+  }, {
+    "path": "https://rawls-dev.broadinstitute.org/listings/workspaces",
```

Comment:
Maybe I'm missing something simple here but I couldn't get the paths to work when they were relative. Because of how swagger looks for the json files, it adds "api-docs" to the front of any relative path that you specify here, so if I were to just do /listings/workspaces, it would become api-docs/listings/workspaces. The only way for me to solve this was to put in the whole path but then this would need to be changed to localhost any time someone launches it locally. Kind of a hassle. Any suggestions?